docs/release-process.md

```diff
@@ -8,73 +24,38 @@
  2. Package two days before a release. So if the release is to be on Thursday,
```

Comment:
To avoid renumbering in the future, we may consider to always use `1.`; a markdown converter should be able to deal with that.